docs/CONTRIBUTING.md

```diff
@@ -1,21 +1,271 @@
+
+* [The list of issues with the **feature definition** label](https://github.com/web-platform-dx/web-features/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+definition%22).
+* [The list of draft features](https://github.com/web-platform-dx/web-features/tree/main/features/draft).
+
+A third case is if you've found a feature that's incorrectly defined. For example if its description is misleading, or you believe the browser support data is incorrect.
+
+In any case, before starting to work on a feature, make sure it isn't already being worked on by checking the list of [pull requests with the **feature definition** label](https://github.com/web-platform-dx/web-features/pulls?q=is%3Aopen+is%3Apr+label%3A%22feature+definition%22).
+
+## Contributing features to the repository
+
+Use the sections below to help you get started with the different ways to contribute to the web-features repository.
+
+### Fork the repository and set up your local environment
+
+Before being able to contribute to the repository, you need to fork it, and get accustomed to the GitHub pull request workflow:
+
+1. Go to the [web-features repository home page](https://github.com/web-platform-dx/web-features).
+1. Click **Fork** in the top-right corner of the page. GitHub creates a fork of the repository under your GitHub account.
+1. Clone the forked repository locally on your computer.
+
+To learn more about the fork and pull request process, see [Fork a repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).
+
+When you have the repository cloned locally, set up your local environment:
+
+1. [Install Node.js](https://nodejs.org).
+
+1. At a command line prompt, navigate to the root of the repository:
+
+   `cd path/to/web-features`
+
+1. Install the project dependencies:
+
+   `npm install`
+
+1. Create a new Git branch to track your work:
+
+   `git checkout -b <your branch name>`
+
+### Create a new feature from scratch
+
+To create a new feature from scratch:
+
+1. Go through the steps in [Fork the repository and set up your local environment](#fork-the-repository-and-set-up-your-local-environment).
+
+1. Decide what the unique ID for your new feature should be. To help you, check the [Identifiers](./guidelines.md#identifiers) section of the feature guidelines.
+
+1. Create a file in the `features` directory with the name of your feature ID, and the `.yml` extension. For example, if your feature ID is `my-feature`, create a file named `features/my-feature.yml`.
+
+1. Open the file in your favorite text editor, and add the following minimum content:
+
+   ```yaml
+   name: "My feature"
+   description: "A short description of the feature."
+   spec: https://urlofthespec.com
+   ```
+
+   For guidance, see the [Names](./guidelines.md#names) and [Descriptions](./guidelines.md#descriptions) sections of the feature guidelines.
```

Comment:
Another place where a highllight could be used.
It would also be cool if we had guidance on `spec`: what to link to, when to have mulitple URLs.